training/data_preparation/README.md

```diff
@@ -0,0 +1,26 @@
+## How to prepare your data for training
+
+Follow the instructions in this document to prepare your data for model training.
+- [Prerequisites](#prerequisites)
+- [Preparing your data](#preparing-your-data)
+
+## Prerequisites
+No special prerequisites other than the standard data ETL tools to create a `.tsv`-formatted file.
+
+## Preparing your data
+The training data must be a sole `.tsv`-formatted file and must carry the `.tsv` extension. 
+Multiple `.tsv` files in the input bucket for training will result in an error.
+
+The file has to be made up of two columns exactly, separated by the tab-delimiter (`\t`) as is typical for a `tsv` file.
+The entry before the `tab`-delimiter must equal a piece of text, and the element after the delimiter must be the class label corresponding to the text before the delimiter. No headers are accepted.
```

Comment:
We need to be more specific. It's not clear what class labels are acceptable. The sample implies that supported labels are `pos` and `neg`. Is `positive` a valid value? The model description indicates that the model is used to measure positive and negative sentiment. What if somebody uses a class label such as `neutral`?

Reply:
I believe the labels can be arbitrary (either binary labels like "positive" or "negative" or more than two labels) -  is that correct @splovyt?

Reply:
Yes it's single-label, multi-class if that makes sense :). So we can have multiple classes such as `very positive`, `positive`, `neutral`, `negative`, `very negative` if we wanted, but only one of those can be assigned at a time. I will make it clear in the docs!